use-cases/NIC-passthrough-and-Kata.md

```diff
@@ -0,0 +1,292 @@
+# How To Pass a Physical NIC Into a Kata Container
+![](https://github.com/Yuval-Ai/documentation/blob/doc-create-NIC-passthrough-and-Kata/use-cases/images/NIC%20passthrough%20Diagram.png)
+
```

Comment:
I wonder if the kata 'default' diagram should be more like the `tc` based one over at https://github.com/kata-containers/documentation/blob/master/design/architecture.md#networking - possibly to show we default to `tc` in the VM and not a virtual nic per-se any more, I think? /cc @amshinde